### ISSUE TITLE
Tensorboard event files syncing not with Google Drive

### ISSUE THREAD

**Issue by F-Schlatt** [googlecolab/colabtools#521]
- Describe the current behavior:
When creating a tensorboard summary file in a mounted google drive, the event files are not automatically synced with google drive. The folder is created and the event files appear locally in the file explorer on google colab. The folder is also synced to google drive, but is empty. Only after manually downloading the file does it appear in google drive and is synced locally. Any other type of file seems to be fine.

- The web browser you are using (Chrome, Firefox, Safari, etc.):
Chrome Version 73.0.3683.103

- Link to self-contained notebook that reproduces this issue
https://colab.research.google.com/drive/1UrFr8QHHKtzmGNNgDbtzVAdK_2VR2_kf

**Comment by colaboratory-team** [comment]
Duplicate of #287

**Comment by pavanteja295** [comment]
Was this issue resolved? flush_and_unmount() function seems to flush but also remove the mounted drive folder. Is there no option to just flush but not unmount ?
